### PR TITLE
Updating UI + Border tokens

### PR DESCRIPTION
**Pull request by corobrie**
fix: 

**Fixes:** # (issue, if applicable)

## Summary:

- Revised border color for focus-visible
- Updated UI-active-on-dark to correct value.

## Type of change:

Please delete options that are not relevant.

- [x] Revision of an existing capability

## Checklist:

- [X] My update follows the CONTRIBUTING guidelines of this project
- [X] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._ 

**Thank you for your submission!**<br>
-- Orion Design System Team

### CODE REVIEW

File: src/color/border.json [hunk 1]
```diff
@@ -66,20 +66,27 @@
         }
       },
       "Focus": {
-        "onLight": {
+        "grey": {
```
Comment (blackfalcon) [inline src/color/border.json]: I am having a hard time swallowing 'grey' in this context. Mainly this descriptive of color in a series of tokens that is mainly use case driven. Also, this is not really "gray" as this is black with an opacity. 

This was previously onLight, so what is the new use case for this border? The token is auro-color-border-focus-gray? This all feels anti-pattern.

File: src/color/border.json [hunk 2]
```diff
@@ -66,20 +66,27 @@
...
           "value": "{color.base.white.value}",
           "public": true,
           "onDark": true,
-          "usage": "Focus border color on dark background",
+          "usage": "Focus border color for dark backgrounds",
           "wcag": "n/a"
-        }
+        },
+        "onLight": {
+          "value": "{color.base.orca.value}",
```
Comment (blackfalcon) [inline src/color/border.json]: I am having growing concerns about using this 'hidden' legacy token for Auro things. It's not fitting the new patten that has been established. https://github.com/AlaskaAirlines/OrionDesignTokens/blob/master/src/color/base.json#L51-L55, 

Currently being used in two other tokens. 
```
color/icon.json
19-      },
20-      "emphasis": {
21-        "onLight": {
22:          "value": "{color.base.orca.value}",

color/text.json
3-    "text": {
4-      "primary": {
5-        "onLight": {
6:          "value": "{color.base.orca.value}",
```

Maybe this is not work for now, but I am wondering if we should consider a black/gray scale of colors as we do with midnight, atlas, breeze, etc ... 

Maybe as example;  
gray-500: #222222
gray-400: #383838
gray-300: #4e4e4e
gray-200: #646464
gray-100: #7a7a7a

File: src/color/border.json [hunk 3]
```diff
@@ -66,20 +66,27 @@
...
+          "public": true,
+          "onDark": true,
```
[the remark this answers is not in the document]
Comment (cobrien64) [reply src/color/border.json]: Agree, Changed

File: src/color/border.json [hunk 4]
```diff
@@ -66,20 +66,27 @@
         }
       },
       "Focus": {
-        "onLight": {
+        "grey": {
           "value": "{color.base.black-opacity-15.value}",
           "public": true,
           "onLight": true,
```
[the remark this answers is not in the document]
Comment (cobrien64) [reply src/color/border.json]: makes sense, Changed

File: src/color/border.json (hunk 2 above)
Comment (cobrien64) [reply src/color/border.json]: LOVEE THIS. Let me play around with the grey concept as a base color. 

I agree, I want to get away from legacy tokens

**Comment by blackfalcon** [comment]
Closing PR. Work is being addressed in PR #23

**Pull request closed without merging**